server/chat-commands/moderation.ts

```diff
@@ -1930,25 +1936,94 @@ export const commands: ChatCommands = {
 		Punishments.addSharedIp(ip, note);
 		note = ` (${note})`;
 
-		this.addGlobalModAction(`The IP '${ip}' was marked as shared by ${user.name}.${note}`);
+		this.privateGlobalModAction(`The IP '${ip}' was marked as shared by ${user.name}.${note}`);
 		this.globalModlog('SHAREDIP', ip, ` by ${user.name}${note}`);
 	},
 	marksharedhelp: [
 		`/markshared [IP], [owner/organization of IP] - Marks an IP address as shared.`,
-		`Note: the owner/organization (i.e., University of Minnesota) of the shared IP is required. Requires @, &`,
+		`Note: the owner/organization (i.e., University of Minnesota) of the shared IP is required. Requires @ &`,
 	],
 
 	unmarkshared(target, room, user) {
 		if (!target) return this.parse('/help unmarkshared');
 		if (!this.can('globalban')) return false;
-		if (!/^[0-9.*]+$/.test(target)) return this.errorReply("Please enter a valid IP address.");
+		if (!IP_REGEX.test(target)) return this.errorReply("Please enter a valid IP address.");
 
 		if (!Punishments.sharedIps.has(target)) return this.errorReply("This IP isn't marked as shared.");
 
 		Punishments.removeSharedIp(target);
 
-		this.addGlobalModAction(`The IP '${target}' was unmarked as shared by ${user.name}.`);
+		this.privateGlobalModAction(`The IP '${target}' was unmarked as shared by ${user.name}.`);
 		this.globalModlog('UNSHAREIP', target, ` by ${user.name}`);
 	},
-	unmarksharedhelp: [`/unmarkshared [ip] - Unmarks a shared IP address. Requires @, &`],
+	unmarksharedhelp: [`/unmarkshared [IP] - Unmarks a shared IP address. Requires @ &`],
+
+	nomarkshared: {
```

Comment:
Since you phrase this 'blacklisting IPs from being marked as shared' in error/help messages, it might make sense to add /marksharedblacklist or something as an alias.